lib/excon/middleware/logger.rb

```diff
@@ -46,6 +46,8 @@ def log_request(datum)
         message += " and took #{duration}ms" if duration
 
         @@sensitive_data.each do |sensitive_data|
+          next unless sensitive_data.present?
```

Comment:
can you also increase the gem patch version ?

Reply:
Done Sr ✅